### PR TITLE
examples: add multi-NLP links

### PR DESCRIPTION
**Pull request by aarnphm**
## What does this PR address?

Add news translator as quickstart examples.

Currently depends on #3543

Signed-off-by: Aaron <29749331+aarnphm@users.noreply.github.com>

## Before submitting:

- [x] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [x] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [x] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [x] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: examples/multi-tasks-nlp/service.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+@svc.api(input=bentoml.io.Text.from_sample(TEXT), output=bentoml.io.Text())
+async def summarize(text: str) -> str:
+    generated = await summarizer.async_run(text, max_length=MAX_LENGTH)
+    return generated[0]["summary_text"]
+
+
+@svc.api(
+    input=bentoml.io.JSON.from_sample({"text": TEXT, "categories": CATEGORIES}),
```
Comment (parano) [inline examples/multi-tasks-nlp/service.py:27]: let's use Pydantic as an example here?

File: examples/multi-tasks-nlp/warmup.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    "healthcare",
+    "parliament",
+    "economy",
+    "infrastructure",
+    "business",
+    "sport",
+    "legal",
+]
+
+CATEGORICAL_THRESHOLD = 0.5
```
Comment (parano) [inline examples/multi-tasks-nlp/warmup.py:41]: how is this defined?

File: examples/multi-tasks-nlp/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+bentoml[all] >= 1.0.16
```
Comment (aarnphm) [inline examples/multi-tasks-nlp/requirements.txt:1]:
```suggestion
bentoml[io-json,grpc,grpc-reflection] >= 1.0.16
```

File: examples/multi-tasks-nlp/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+torch
+transformers
+requests
+pydantic
```
Comment (aarnphm) [inline examples/multi-tasks-nlp/requirements.txt:5]:
```suggestion
```

**Pull request closed without merging**